inductiva/molecules/scenarios/protein_solvation/post_processing.py

```diff
@@ -65,12 +71,12 @@ def construct_universe(self, use_compressed_trajectory: bool = False):
             or the full precision trajectory."""
         topology = os.path.join(self.sim_output_dir, "solvated_protein.tpr")
         if use_compressed_trajectory:
-            trajectory = os.path.join(self.sim_output_dir, "trajectory.xtc")
+            trajectory = os.path.join(self.sim_output_dir,
+                                      "compressed_trajectory.xtc")
         else:
             trajectory = os.path.join(self.sim_output_dir,
-                                      "solvated_protein.trr")
-
-        universe = unwrap_trajectory(topology, trajectory)
+                                      "full_trajectory.trr")
```

Comment:
should these filenames be defined as constants somewhere?

Reply:
Yes, that makes sense. Maybe as global variables?

Reply:
Yes. Pls do that and then we will figure out how to manage these better.
But are they constants of the scenario? It seems so...